packages/rulesets/src/spectral/functions/delete-response-codes.ts

```diff
@@ -15,13 +17,38 @@ export const DeleteResponseCodes = (deleteOp: any, _opts: any, ctx: any) => {
   const path = ctx.path
   const errors = []
 
-  if (!deleteOp?.responses) {
-    return []
+  if (!deleteOp?.responses || deleteOp?.responses.length == 0) {
+    errors.push({
+      message: EmptyResponse_ERROR,
+      path: path,
+    })
+    return errors
   }
 
   const responses = Object.keys(deleteOp?.responses ?? {})
```

Comment:
I think it should be this now that I'm seeing this line again

```suggestion
  const responses = Object.keys(deleteOp?.responses ?? [])
```